web/libs/datamanager/src/sdk/lsf-sdk.js

```diff
@@ -136,11 +136,14 @@ export class LSFWrapper {
     if (this.datamanager.hasInterface("autoAnnotation")) {
       interfaces.push("auto-annotation");
     }
-    if (this.interfacesModifier) {
-      interfaces = this.interfacesModifier(interfaces, this.labelStream);
-    }
+
     if (isFF(FF_DEV_2887)) {
       interfaces.push("annotations:comments");
+      interfaces.push("comments:resolve-any");
```

Comment:
why do we need to set this flag in LSO if we don't want this to be in LSO?

Reply:
It allows to work with comments as it was before, without roles affects them.